docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+
+1. The ngrok agent in your network connects to one or more of your upstream services.
+2. The ngrok agent connects out to the ngrok network at the default [agent ingress address](/docs/agent/ingress/) at `connect.ngrok-agent.com` (or a custom one).
+3. The ngrok network creates an endpoint like `your-company.your-vendor.com`, which then points to your upstream service.
+4. Your vendor sends requests from their services to the endpoint, which are then routed through to the agent and finally your upstream service.
+
+### What is ngrok’s network architecture?
+
+We run ngrok’s primary network and services on AWS.
+
+This architecture includes multiple [Points of Presence](/docs/network-edge/#points-of-presence) (PoPs) worldwide that run our services, route customer traffic, and make up our always-on [Global Server Load Balancer](https://ngrok.com/blog-post/gslb-global-server-load-balancing) (GSLB).
+
+The GSLB, which is active by default for all ngrok endpoints and tunnels, enhances the resiliency of the connection between your services and your vendor’s with DDoS protection and geo-failover that automatically routes traffic to the lowest-latency PoP in the event of downtime or service interruption. If you have specific data residency requirements, like those located in the EU, you and your vendor can configure ngrok to use a specific region in Europe or pin all traffic to a specific PoP.
+
+There are many possible architectures for setting up a site-to-site network based on the shared requirements of you and your vendor. Your vendor will work with you to find a mutually secure and reliable architecture.
+
+### How does ngrok handle data residency requests?
+
+ngrok is user-configurable to match your data residency needs.
+
+First, [configure the agent](/docs/guides/upgrade-v2-v3/#changes-to-choosing-a-region) to use a PoP in one of our [supported regions](/docs/network-edge/#points-of-presence). Next, work with your vendor to set up appropriate DNS to route all connections through the same data plane.
+
+Our regional data planes are located in Australia (Sydney), Europe (Frankfurt), India (Mumbai), Japan (Tokyo), South America (São Paulo), United States (California and Ohio).
+
+### Is ngrok a dedicated instance or multi-tenant?
+
+ngrok is a multi-tenant application with services shared across our customer base, including your vendor and your site-to-site architecture.
+
+## Security
+
+### How can I ensure ngrok does not pose a security risk to my infrastructure?
+
+We recommend you begin by exploring our [Security, Privacy, and Compliance](https://ngrok.com/security) page, followed by our [Trust Center](https://trust.ngrok.com/).
+
+ngrok is deeply configurable to enforce your established security policies, including:
+
+- [Prevent unauthorized usage of ngrok](#how-can-i-prevent-ngrok-from-being-used-for-any-purpose-other-than-in-site-to-site-connectivity-with-my-vendor?) not outside of the site-to-site connectivity with your vendor.
```

Comment:
[Prevent unauthorized usage of ngrok](#how-can-i-prevent-ngrok-from-being-used-for-any-purpose-other-than-in-site-to-site-connectivity-with-my-vendor?) outside of the site-to-site connectivity with your vendor.

I think *not* is not supposed to be there.